Code/Project/Project/Task.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Project
+{
+    public class Task
+    {
+        private string _Name = "";
+        private string _Description = "";
+        private int _Rating;
+
+        public string Name { get { return _Name; } set { _Name = value; } }
+        public string Description { get { return _Description; } set { _Description = value; } }
+        public int Rating { get { return _Rating; } set { _Rating = value; } }

```

Comment:
You dont need the private versions of the vars. "pulbic Type Var" would be enough.

Reply:
sure